lib/shared/log/tasks.js

```diff
@@ -71,55 +77,65 @@ function logTasks(tree, opts, getTask) {
   }
 
   function addTaskToLines(task, lines, isLast, isLeaf) {
-    var taskBars = task.bars + (isLast ? '└' : '├') + '─';
+    var taskBars = task.bars + (isLast
+        ? translate.message(messages.BOX_DRAWINGS_LIGHT_UP_AND_RIGHT)
+        : translate.message(messages.BOX_DRAWINGS_LIGHT_VERTICAL_AND_RIGHT)) +
+          translate.message(messages.BOX_DRAWINGS_LIGHT_HORIZONTAL);
     if (isLeaf) {
-      taskBars += '─ ';
+      taskBars += translate.message(messages.BOX_DRAWINGS_LIGHT_HORIZONTAL);
     } else {
-      taskBars += '┬ ';
+      taskBars += translate.message(messages.BOX_DRAWINGS_LIGHT_DOWN_AND_HORIZONTAL);
     }
+    taskBars += ' ';
 
     var line = {};
     if (task.depth === 1) {
-      line.label = chalk.white(taskBars) + chalk.white(task.label);
+      line.label = taskBars + task.label
     } else {
-      line.label = chalk.white(taskBars) + chalk.cyan(task.label);
+      line.label = taskBars + task.label;
     }
+    line.width = stringWidth(line.label);
+
     if (typeof task.desc === 'string' && task.desc) {
-      line.desc = chalk.white(task.desc);
+      line.desc = task.desc;
     }
     lines.push(line);
 
-    var maxLabelWidth = line.label.length
+    var maxLabelWidth = line.width;
 
     if (!isObject(task.flags)) {
       return maxLabelWidth;
     }
 
     var flagBars = task.bars;
     if (isLast) {
-      flagBars += '  ';
+      flagBars += ' ';
     } else {
-      flagBars += '│ ';
+      flagBars += translate.message(messages.BOX_DRAWINGS_LIGHT_VERTICAL);
     }
+    flagBars += ' ';
 
     if (isLeaf) {
-      flagBars += '  ';
+      flagBars += ' ';
     } else {
-      flagBars += '│ ';
+      flagBars += translate.message(messages.BOX_DRAWINGS_LIGHT_VERTICAL);
     }
+    flagBars += ' ';
 
     Object.entries(task.flags).sort(flagSorter).forEach(addFlagsToLines);
 
     function addFlagsToLines(ent) {
       if (typeof ent[0] !== 'string' || !ent[0]) return;
       var line = {};
-      lines.push(line);
-      line.label = chalk.white(flagBars) + chalk.magenta(ent[0]);
```

Comment:
Same as above. No longer allowing theming here.

Reply:
This depends on https://github.com/gulpjs/gulp-cli/pull/260#discussion_r1528380052

Reply:
Can you explain?

Reply:
I'm starting to think these need to pass through `translate.message` because it's annoying for users to theme the keys on `task.flags = {}`

Reply:
@sttk I've added more messages so I can translate this.